android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -369,8 +370,10 @@ synchronized public void setNowPlaying(ReadableMap metadata) {
         nb.setContentInfo(album);
         nb.setColor(notificationColor);
         nb.setColorized(false);
-
-        notification.setCustomNotificationIcon(notificationIcon);
+ 
+        if(notificationIcon != null){
```

Comment:
You've already added `if (notification == null) return;` at the start of this function (line 325), so this check here is not unnecessary, right?

Reply:
this is for `notificationIcon`. so if it is null then we should not even let it execute unless it is a URL found from setnowplaying. so I think this should be good.